### PR TITLE
intel-gpu-tools: update and rename to igt-gpu-tools

### PR DESCRIPTION
**Pull request by m3tav3rse**

### CODE REVIEW

File: srcpkgs/igt-gpu-tools/template [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+version=1.23
+revision=1
+build_style=meson
+hostmakedepends="flex gtk-doc pkg-config swig"
+makedepends="cairo-devel libXrandr-devel libXv-devel libkmod-devel
+ libunwind-devel procps-ng-devel python3-devel libressl-devel"
+short_desc="Tools for development and testing of the Intel DRM driver"
+maintainer="Juan RP <xtraeme@voidlinux.eu>"
+license="MIT"
+homepage="https://cgit.freedesktop.org/xorg/app/intel-gpu-tools/"
```
Comment (CameronNemo) [inline srcpkgs/igt-gpu-tools/template]: You could probably change the homepage over to https://gitlab.freedesktop.org/drm/igt-gpu-tools if you want.

Comment (m3tav3rse) [reply srcpkgs/igt-gpu-tools/template]: Good idea, didn't realize cgit is mirror now.

File: srcpkgs/igt-gpu-tools/template [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+lib32disabled=yes
+only_for_archs="i686 x86_64"
+
+post_install() {
+	vlicense COPYING
+}
+
+intel-gpu-tools_package() {
+	depends="igt-gpu-tools"
+	short_desc="Transition package for intel-gpu-tools"
```
Comment (maxice8) [inline srcpkgs/igt-gpu-tools/template]: please put

```
short_desc+=" (transitional dummy package)"
```

it is more commonly used

File: srcpkgs/igt-gpu-tools/template [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+
+lib32disabled=yes
+only_for_archs="i686 x86_64"
+
+post_install() {
+	vlicense COPYING
+}
+
+intel-gpu-tools_package() {
+	depends="igt-gpu-tools"
```
Comment (maxice8) [inline srcpkgs/igt-gpu-tools/template]: igt-gpu-tools>=${version}_${revision}

**Pull request closed without merging**